includes/controls/media.php

```diff
@@ -64,11 +64,11 @@ public function get_default_value() {
 	 * @return array Control settings.
 	 */
 	public function on_import( $settings ) {
-		if ( empty( $settings['url'] ) ) {
+		if ( empty( $settings['url'] ) || strpos( $settings['url'], 'images/placeholder.png' ) !== false ) {
```

Comment:
Hi, thanks for the suggestion!

Can you please move the check to the `import` method that is responsible for the import?
`Plugin::$instance->templates_manager->get_import_images_instance()->import( $settings );`

Also, the check need to be more specific `elementor/assets/images/placeholder.png`.

Reply:
Thanks for your observation. I'll make the changes and commit. After that I'll let you know here.

Reply:
I've adopted the changes requested.